packages/jest-preset/helpers/__tests__/babel-plugin-import-component.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-env node, jest */
+
+const { resolve, join } = require('path');
+const { readdirSync: readdir, readFileSync: readFile } = require('fs');
+const babel = require('@babel/core');
+
+const fixtureFolder = resolve(__dirname, 'fixture');
+const file = (type, count) =>
+  readFile(join(fixtureFolder, `${type}-${count}.txt`), 'utf8').trim();
+const fixtures = readdir(fixtureFolder).filter(file => file.endsWith('.txt'));
+const tests = fixtures
+  .filter(file => file.startsWith('code'))
+  .map(file =>
+    file
+      .split('.')
+      .shift()
+      .split('-')
+      .pop()
+  )
+  .map(count => [count, file('code', count), file('expected', count)]);
+
+const babelOptions = {
+  plugins: [
+    require.resolve('../babel-plugin-import-component'),
+    require.resolve('@babel/plugin-syntax-dynamic-import'),
+  ],
+  babelrc: false,
+};
+
+describe('babel-plugin-import-component', () => {
+  it.each(tests)(
+    'should transpile the code correctly (fixture %s)',
+    (_, code, expected) => {
+      const result = babel.transform(code, babelOptions).code.trim();
+
+      expect(result).toBe(expected);
+    }
+  );
+});
```

Comment:
What about totally dumb implementation of the same tests? It does the same but **zero magic and fun** involved. 🦊 

```js
/* eslint-env node, jest */
const { resolve, join } = require('path');
const { readFileSync: readFile } = require('fs');
const babel = require('@babel/core');

const fixtureFolder = resolve(__dirname, 'fixture');
const tests = [
  ['code-01.txt', 'expected-01.txt'],
  ['code-02.txt', 'expected-02.txt'],
  ['code-03.txt', 'expected-03.txt'],
  ['code-04.txt', 'expected-04.txt'],
];

const babelOptions = {
  plugins: [
    require.resolve('../babel-plugin-import-component'),
    require.resolve('@babel/plugin-syntax-dynamic-import'),
  ],
  babelrc: false,
};

describe('babel-plugin-import-component', () => {
  it.each(tests)(
    'should transpile the code correctly (fixture %s)',
    (codeSource, expectedSource) => {
      const code = readFile(join(fixtureFolder, codeSource), 'utf8');
      const expected = readFile(join(fixtureFolder, expectedSource), 'utf8');
      const result = babel.transform(code, babelOptions).code;

      expect(result.trim()).toBe(expected.trim());
    }
  );
});

```